amkotlinutil/src/main/java/dk/adaptmobile/amkotlinutil/extensions/ViewAnimationExtensions.kt

```diff
@@ -288,6 +290,22 @@ fun View.fadeInUp(duration: Long = 250, offset: Float? = null) {
         this.translationY = offset ?: this.height.toFloat() - (this.height / 2)
         this.alpha = 0f
         this.visible()
-        this.animate(true).translationY(0f).alpha(1f).setDuration(duration).setInterpolator(AccelerateDecelerateInterpolator())
+        this.animate(true).translationY(0f).alpha(1f).setDuration(duration).interpolator = AccelerateDecelerateInterpolator()
```

Comment:
I think you could leave that as it was before: .setDuration(duration).setInterpolator(AccelerateDecelerateInterpolator())
Have you tested that the function work?

Reply:
Have not tested that function, but it should be doing the same? The end is setting the interpolator with AccelerateDecelateInterpolator() in both cases.